test/factory.js

```diff
@@ -0,0 +1,34 @@
+const faker = require('faker');
+
+module.exports.createUser = () => {
+  const first = faker.name.firstName(),
+    last = faker.name.lastName(),
+    roleID = 2;
+  const fakeUser = {
+    username: faker.internet.userName(),
+    name: { first, last },
+    email: faker.internet.email(),
+    password: faker.internet.password(),
+    roleId: roleID
+  };
+  return fakeUser;
+};
+
+module.exports.createDocument = () => {
+  const fakeDocument = {
+    published: Date(),
+    title: faker.lorem.word(),
+    access: 'public',
+    content: faker.lorem.sentences(),
+    ownerId: 2,
+    ownerRoleId: 2
+  };
+  return fakeDocument;
```

Comment:
**Code Quality - Nitpick**
You can export JavaScript object as modules, no need to wrap a function around it.
```javaScript
module.exports.createDocument = {
  published: Date(),
  title: faker.lorem.word(),
  access: 'public',
  content: faker.lorem.sentences(),
  ownerId: 2,
  ownerRoleId: 2
};
```